src/messages/index.js

```diff
@@ -1,13 +1,6 @@
 import "./messages.css";
 
-const Messages = ({ checkWin, visible, isCorrect }) => {
-  if (checkWin !== 0) {
-    visible = true;
-  }
-  if (checkWin === 2) {
-    isCorrect = true;
-  }
-
+const Messages = ({ visible, isCorrect, newQuestion }) => {
   if (visible) {
```

Comment:
Do the bit flipping we talked about the other day. It's better to say `if (!visible) return null` than to get too many braces involved

Reply:
So instead of:
```js
if (visible) {
 // do something
} else {
  return null;
}
```

It's simpler and easier to do

```js
if (!visible) return null;

// do something
```